cross-linux-android/hooks/images.csv

```diff
@@ -0,0 +1,4 @@
+DOCKERFILE;ARGS (comma-separated list);TAG
+;NDK=19;ndk19
+;NDK=20;ndk20
+;NDK=21;ndk21
```

Comment:
I'd start with just two NDK versions: the current LTS and the oldest version we would like to support. Do you have a recommendation on such a version? If not, let's start with just ndk21 for now :)

Reply:
Sorry I've no suggestion atm for the oldest version to use; So ok I'll change to build only `ndk21` image.